addons/mail/wizard/mail_wizard_invite.py

```diff
@@ -61,23 +43,51 @@ def add_followers(self):
             document.message_subscribe(partner_ids=new_partners.ids)
 
             model_name = self.env['ir.model']._get(wizard.res_model).display_name
-            # send a notification if option checked and if a message exists (do not send void notifications)
-            if wizard.notify and wizard.message and not is_html_empty(wizard.message):
-                message_values = wizard._prepare_message_values(document, model_name, email_from)
-                message_values['partner_ids'] = new_partners.ids
-                document.message_notify(**message_values)
+            message_values = wizard._prepare_message_values(document, model_name, user_from)
+            message_values['partner_ids'] = wizard.partner_ids.ids
+            message_values['subtype_xmlid'] = "mail.mt_comment"
+            document.message_notify(**message_values)
         return {'type': 'ir.actions.act_window_close'}
 
-    def _prepare_message_values(self, document, model_name, email_from):
+    def _prepare_message_values(self, document, model_name, user_from):
+        inviter_image_url = f"/web/image/res.partner/{user_from.partner_id.id}/avatar_128"
+        creator_image_url = f"/web/image/res.users/{document.create_uid.id}/avatar_128"
+        body = Markup("""
+            <div style="display:flex;">
+                %(inviter_avatar)s%(inviter_name)s (%(email_from)s) %(content)s
+            </div>
+            <br/>
+            <div style="font-size:13px;border:1px solid grey;border-radius:5px;padding:5px;width:fit-content">
+                %(record_name)s (%(model_description)s)
+                <div style="display:flex;padding-top:5px;">
+                    Created on "%(created_on)s", by &nbsp; %(creator_avatar)s%(created_by)s
+                </div>
+            </div>
+            <div style="color:grey">
+                %(message)s
+            </div>
+        """) % {
+            'created_by': document.create_uid.display_name,
+            'created_on': datetime.date(document.create_date).strftime('%m-%d-%Y'),
+            'creator_avatar': Markup("<img style='height:20px;width:20px;margin-right:5px;' alt='Avatar' src=%s/>") % creator_image_url,
+            'content': _("added you as a follower to this document"),
```

Comment:
Would use a complete sentence to ease translations, like 
```suggestion
            'content': _("%(inviter_name)s added you as a follower to this document", inviter_name=...),
```